### ISSUE TITLE
crash with django debugview

### ISSUE THREAD

**Issue by jessewgibbs** [rollbar/pyrollbar#180]
Details in https://app.clubhouse.io/rollbar/story/730/pyrollbar-crash-with-django-debugview

**Comment by mjhanke** [comment]
@jessewgibbs Could the details of this issue please be made public, and not hidden behind a login and requiring us to be in the rollbar project? Thanks.

**Comment by brianr** [comment]
Hi @mjhanke , are you experiencing this same issue?

I tracked down the original thread about this... it was from 2015. Here was the relevant stack trace there:

```
2015-04-22 19:21:11,355 ERROR [rollbar.contrib.django.middleware][MainThread] Exception while adding view-in-rollbar link to technical_500_template.

Traceback (most recent call last):
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/rollbar/contrib/django/middleware.py", line 66, in new_get_traceback_data
item_uuid = exception_reporter.request.META.get('rollbar.uuid')
AttributeError: 'NoneType' object has no attribute 'META'
```

**Comment by mjhanke** [comment]
We suspect we might, but looking at that traceback, I don't think so. I thought there was a chance since this issue was referenced by #215, which matches the traceback we're seeing. However, we're running rollbar 0.14.5, which included the fix for #215. We _are_ also running the django debug toolbar, which is why I thought perhaps this issue may explain what we're seeing.

This is the traceback we're seeing, which is almost exactly like #215 
```
[ERROR]	2019-09-17T04:19:55.506Z	d8024161-f0a6-4e42-b421-dbdf8ad4cb6c	Exception while posting item SSLError(MaxRetryError("HTTPSConnectionPool(host='api.rollbar.com', port=443): Max retries exceeded with url: /api/1/item/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:852)'),))",),)
Traceback (most recent call last):
  File "/var/task/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/var/task/urllib3/connectionpool.py", line 346, in _make_request
    self._validate_conn(conn)
  File "/var/task/urllib3/connectionpool.py", line 850, in _validate_conn
    conn.connect()
  File "/var/task/urllib3/connection.py", line 326, in connect
    ssl_context=context)
  File "/var/task/urllib3/util/ssl_.py", line 329, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/var/lang/lib/python3.6/ssl.py", line 407, in wrap_socket
    _context=self, _session=session)
  File "/var/lang/lib/python3.6/ssl.py", line 817, in __init__
    self.do_handshake()
  File "/var/lang/lib/python3.6/ssl.py", line 1077, in do_handshake
    self._sslobj.do_handshake()
  File "/var/lang/lib/python3.6/ssl.py", line 689, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:852)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/var/task/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/var/task/urllib3/util/retry.py", line 388, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.rollbar.com', port=443): Max retries exceeded with url: /api/1/item/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:852)'),))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/rollbar/__init__.py", line 1336, in _send_payload
    _post_api('item/', payload_str, access_token=access_token)
  File "/var/task/rollbar/__init__.py", line 1385, in _post_api
    proxy_password=SETTINGS.get('http_proxy_password'))
  File "/var/task/rollbar/lib/transport.py", line 33, in post
    return _session().post(*args, proxies=proxies, **kw)
  File "/var/task/requests/sessions.py", line 555, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/var/task/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/var/task/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/var/task/requests/adapters.py", line 506, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='api.rollbar.com', port=443): Max retries exceeded with url: /api/1/item/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:852)'),))
```

**Comment by brianr** [comment]
Got it - yeah, that looks like the same as #215. I'll reopen that issue.
